baybe/parameters/substance.py

```diff
@@ -22,6 +23,9 @@
     if _MORDRED_INSTALLED:
         from baybe.utils import smiles_to_mordred_features
 
+Smiles = str
```

Comment:
Having just the first letter capitalized looks weird to me, is this how a type alias should be defined?

Reply:
Yup, the convention for aliases is the same as for classes: CamelCase. At least that's my state of knowledge

Reply:
Ok, then I'll trust you on this ;)